wrappers/pointcloud/pointcloud-stitching/doc/pointcloud-stitching-demo.md

```diff
@@ -0,0 +1,189 @@
+ 
+
+![rs-pointcloud-stitching ir only](https://user-images.githubusercontent.com/40540281/127966989-2474f2ab-475b-47be-bc16-547f3d2f43c9.png)
+
+
+Use the button marked **"Save Frames"** to save frames of the checkerboard target in different locations, according to the calibration program's needs (10 – 20 images according to MATLAB's® "Stereo Camera Calibrator"® guide)</br>
+The images are saved in 2 separate folders, 1 for each camera, under the given `<working directory>/calibration_input` directory.</br>
+In our case, they are saved here: </br> `C:\pc_stitching_ws\calibration_input`</br>
+In addition, the application also creates 2 files in the calibration_input directory, describing the intrinsics of the cameras.
+
+### __Calculating calibration matrix using MATLAB's® "Stereo Camera Calibrator"®__
+- Open MATLAB and set directory to your `librealsense\wrappers\pointcloud\pointcloud-stitching` directory.
+- Load intrinsics parameters using the given script:
+```
+>> intrin_831 = load_intrinsics("C:\pc_stitching_ws\calibration_input\intrinsics_831612073525.txt");
+>> intrin_912 = load_intrinsics("C:\pc_stitching_ws\calibration_input\intrinsics_912112073098.txt");
+```
+- Launch stereoCameraCalibrator:
+```
+>> stereoCameraCalibrator
+```
+- Inside stereoCameraCalibrator:
+    1. Use “Add Images” button to load the saved images:
+        - The images were saved in the working directory under "calibration_input/ <serial_number>".
+        - The app calculates calibration from camera1 to camera2. Remember which device you set for which index.
+        - Measure the size of a square on target – in our case: 36.8mm, and set the value in the "Size of checkerboard square" text box.
+    - Load Intrinsics using the "Use fixed intrinsics->Load Intrinsics” button.
+    - Use “Calibrate” button to calculate the calibration values.
+    - Use “Export camera parameters” to export to workspace.
+
+![stereoCameraCalibrator](https://user-images.githubusercontent.com/40540281/127968760-c0d9d5cd-2416-45cc-ab94-c5e698768f3d.png)
+ 
+### __Converting calibration matrix to rs-pointcloud-stitching's units.__
+- Use the provided MATLAB function to create an rs-pointcloud-stiching calibration file from the calibration results:
+
+```
+export_calibratrion(912112073098, 831612073525, stereoParams, 'C:\pc_stitching_ws\calibration_60m.cfg')
+```
+**Notice:**
+- The first given serial number belongs to camera1 and the second to camera2, as specified using the "Add Images" button in stereoCameraCalibrator. **The order is important.**
+- stereoParams – the parameter exported by stereoCameraCalibrator
+- 'C:\pc_stitching_ws\calibration_60m.cfg' – The output calibration file.
+
+The function will create the calibration file needed for rs-pointcloud-stitching based on the calculated calibration. It will also place the united camera in the middle between the 2 input devices.

```

Comment:
` It will also place the united camera` - pls rephrase it as at this stage the Virtual camera concept is not introduced yet